go.mod

```diff
@@ -84,7 +98,7 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/api v0.28.3 // indirect
+	k8s.io/api v0.28.3
```

Comment:
Maybe run `go mod tidy`

Reply:
Because that k8s api should bé in the direct require section AT thé top

Reply:
I did `go mod tidy` but the require stays in that section!
I'm doing the move manually...